Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -80,6 +80,9 @@ open class ChatMessageListVC:
 
         return !listView.isLastCellFullyVisible && isMoreContentThanOnePage
     }
+
+    /// A callback closure that is called when the message list is tapped
+    open var didTapOnMessageListView: ((UITapGestureRecognizer) -> Void)?
```

Comment:
hmm, why not a new delegate method? 🤔

Reply:
ok, found the answer in the description 😄

Reply:
I would personally go with a new delegate method + empty implementation since we already handle all the actions from the message list this way. But leaving it's up to you, other things look solid 👍

Reply:
I've mentioned this in the PR description. It is kinda overkill to add a new delegate event for this and will add maintenance costs. Because we will need to add an empty implementation to this new delegate entry so that we don't break the API. IMO It is not worth it, even though the delegate would be more consistent. Not sure on this one. I feel that the delegates are not that great in SDK, because they easily break the API, unless we provide default implementation, which most of the cases it causes more problems.